infiniband-diags/man/infiniband-diags.8.in.rst

```diff
@@ -99,8 +99,8 @@ The following config files are common amongst many of the utilities.
 Utilities list
 ==============
 
-Basic fabric conectivity
-------------------------
+Basic fabric connectivity
```

Comment:
We should also run lintian from the CI...

Reply:
lintian operates on Debian packages. So it could run after a Debian package build. lintian learns new spelling mistakes from time to time.